### ISSUE TITLE
Make the image available for team to run offline

### ISSUE THREAD

**Issue by mduaneh** [wpilibsuite/DetectCoral#19]
Is it possible to make the s3 buckets and image public so teams can learn how the data from supervise.ly is ingested into the sagemaker model

**Comment by GrantPerkins** [comment]
If you go to the dev branch, you will see the files used to make the ECR/Docker image. In early development, we manually built our Docker image on a laptop running Ubuntu, and trained locally. However, this is a lot of work and it's not really reasonable for teams to do it. If you are motivated to get this running locally, I am more than happy to help you get it up and running.

We are currently working on getting an alternative to AWS Educate up and running, however. This explains why a local version is not officially in development.

**Comment by mduaneh** [comment]
Awesome!  Thanks!  Unless the AWS alternative will be arriving by next week we have to try an alternative. I am a mentor and trying to setup all this locally so the kids are in roughly the same spot as if AWS  was working. I am also trying to create something very similar to the rPi image on a Jetson nano. So far I think I bit off more than I can chew

**Comment by GrantPerkins** [comment]
The AWS Educate alternative is out of my control, and I do not know when it will come into effect, hopefully asap.

The only thing you should have to do is build/start the Dockerfile, and run the training script. You will need to change the paths of where the data is, where scripts are, and where the output should go, but the hard work is done for you. The script for converting data from Supervise.ly .tar.gz to TensorFlow record is there, the script that does transfer learning with a mobilenet V2 is there as well.

Making a rPi image for the Jetson nano was once upon a time on my todo list, but RPI+Google Coral was too easy/good for the team to pass up.

**Comment by mduaneh** [comment]
I think I have a good starting point here.  I'm using a cpu only to train and its definitely taking a toll on a single CPU.
I had to make some updates to the Dockerfile as I kept getting various pip or missing package errors.  One thing I found confusing:  Why is there switching between python2 and python3 (pip/pip3) in several cases?

Here is how I started the container:
docker run -it --rm -p 8888:8888 -v $PWD:/opt/ml -w /opt/ml -v $(realpath ../data):/opt/ml/input/data/training frc-training:v1.0 "$@"
